src/lib/es2021.intl.d.ts

```diff
@@ -21,4 +21,106 @@ declare namespace Intl {
         dayPeriod?: "narrow" | "short" | "long";
         fractionalSecondDigits?: 0 | 1 | 2 | 3;
     }
+
+    /**
+     * The locale matching algorithm to use.
+     *
+     * [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/ListFormat/ListFormat#parameters).
+     */
+    type ListFormatLocaleMatcher = "lookup" | "best fit";

```

Comment:
One final comment: I'm noticing that none of the options above have defined types (or make use of the new ones); is that intentional? Is it more "correct" to not have defined types for any of them? Or do they need to be fixed to have defined types too?

(I'm also noticing that there are some silly things above like `"best fit" | "best fit"`.)

Reply:
I'd rather not touch it in this PR, but create a follow-up PR in the next days instead.

* Create named types for the options above, remove duplicate `"best fit"`
* Investigate if `ResolvedDateTimeFormatOptions` is even correct (at a first glance, it seems wrong, but might be to match some unique browser behavior?)

Feel free to disagree, I can also tackle them here

Reply:
I'm sure a second PR is fine, I just wasn't sure (I have no major knowledge of these types) whether or not it's normal to have top level declared type names or not.